app/__init__.py

```diff
@@ -19,13 +20,14 @@ def create_app(config_name):
                         static_folder='static/',
                         static_url_path=configs[config_name].STATIC_URL_PATH)
 
-    application.config.from_object(configs[config_name])
-    configs[config_name].init_app(application)
-    config.init_app(application)
-
-    bootstrap.init_app(application)
-    logging.init_app(application)
-    data_api_client.init_app(application)
+    init_app(
+        application,
+        configs[config_name],
+        bootstrap=bootstrap,
+        data_api_client=data_api_client,
+        feature_flags=feature_flags
+    )
+    # Should be incorporated into digitalmarketplace-utils as well
     csrf.init_app(application)
```

Comment:
This is a bit sucky -- basically shows how brittle our [`init_app`](https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/__init__.py#L8) function is. 
Would it be better to generalize `init_app()` in dmutils so that it could take a dictionary of objects and call `.init_app(application)` on each of them?